tests/test_pyansys_logging.py

```diff
@@ -0,0 +1,141 @@
+        test_logger.error("Error stdout with level=INFO")
+        test_logger.critical("Critical stdout with level=INFO")
+
+        # Modify the level
+        test_logger.setLevel(level=logging.WARNING)
+        test_logger.debug("Debug stdout with level=WARNING")
+        test_logger.info("Info stdout with level=WARNING")
+        test_logger.warning("Warning stdout with level=WARNING")
+        test_logger.error("Error stdout with level=WARNING")
+        test_logger.critical("Critical stdout with level=WARNING")
+
+    # level=INFO
+    assert "DEBUG -  - test_pyansys_logging - test_level_stdout - Debug stdout with level=INFO" not in capture.content
+    assert "INFO -  - test_pyansys_logging - test_level_stdout - Info stdout with level=INFO" in capture.content
+    assert "WARNING -  - test_pyansys_logging - test_level_stdout - Warning stdout with level=INFO" in capture.content
+    assert "ERROR -  - test_pyansys_logging - test_level_stdout - Error stdout with level=INFO" in capture.content
+    assert "CRITICAL -  - test_pyansys_logging - test_level_stdout - Critical stdout with level=INFO" in capture.content
+    # level=WARNING
+    assert "INFO -  - test_pyansys_logging - test_level_stdout - Info stdout with level=WARNING" not in capture.content
+    assert (
+        "WARNING -  - test_pyansys_logging - test_level_stdout - Warning stdout with level=WARNING" in capture.content
+    )
+    assert "ERROR -  - test_pyansys_logging - test_level_stdout - Error stdout with level=WARNING" in capture.content
+    assert (
+        "CRITICAL -  - test_pyansys_logging - test_level_stdout - Critical stdout with level=WARNING" in capture.content
+    )
+
+    # File handlers are not activated.
+    assert os.path.exists(os.path.exists(os.path.join(os.getcwd(), "PyProject.log")))
+
+
+def test_default_file_handlers():
+    """Activate the `PyProject.log` file handler."""
+
+    current_dirctory = os.getcwd()
+    file_logger = os.path.join(current_dirctory, "PyProject.log")
+    if os.path.exists(file_logger):
+        os.remove(file_logger)
+
+    content = None
+    test_logger = pyansys_logging.Logger(to_file=True)
+    test_logger.info("Test PyProject.Log")
+
+    with open(file_logger, "r") as f:
+        content = f.readlines()
+
+    assert len(content) == 6
+    assert "NEW SESSION" in content[2]
+    assert "===============================================================================" in content[3]
+    assert "LEVEL - INSTANCE NAME - MODULE - FUNCTION - MESSAGE" in content[4]
+    assert "INFO -  - test_pyansys_logging - test_default_file_handlers - Test PyProject.Log" in content[5]
+
+    # Remove file's handlers and delete the file.
+    for handler in test_logger.logger.handlers:
+        if isinstance(handler, logging.FileHandler):
+            handler.close()
+            test_logger.logger.removeHandler(handler)
+    os.remove(file_logger)
+
+
+def test_file_handlers():
```

Comment:
Use the [tmpdir](https://docs.pytest.org/en/6.2.x/tmpdir.html) fixture. We want to avoid creating temporary files because these can easy fail to be cleaned up should the test fail for some reason. `tmpdir` creates temporary files in the temp directory (OS dependent) and will not be persistent.

Reply:
Thanks @akaszynski for the suggestion. I removed the `os.removed(file_logger)` because in the doc of the `tmpdir` fixture, it is said that it will delete the former pytest tempdir after three testing session. I checked and indeed it is the behavior.
![image](https://user-images.githubusercontent.com/87315832/148040102-c32b35ea-d812-4580-84a6-beef75b1ac3d.png)